test/unit/vl-map-search.test.html

```diff
@@ -205,6 +322,207 @@
         assert.equal(element.getTranslation('select.no_more_options'), text4);
       });
     });
+
+    suite('Lambert-coördinaat', () => {
+      test('of - een vrije tekst, geeft "undefined"', () => {
```

Comment:
Waarvoor staat `of - ` in de test beschrijving?

Reply:
functienaam "-" case ...

Reply:
maar had ik denk ik ook al opgemerkt dat dat anders moet